planning/behavior_path_goal_planner_module/src/goal_planner_module.cpp

```diff
@@ -39,7 +39,7 @@
 #include <utility>
 #include <vector>
 
-using behavior_path_planner::utils::start_goal_planner_common::calcFeasibleDecelDistance;
+using behavior_path_planner::utils::parking_departure::calcFeasibleDecelDistance;
```

Comment:
what is rename to parking_depature ?

Reply:
I don't want to put planner specific utils in `behavior_path_planner_common` package. In my understanding, there is a place to put general util functions or interfaces.

But in this case, since start and goal planner uses common utils, he need to put in in `behavior_path_planner_common` package I guess. So he rename the utils file name to show that the utils file is **NOT** module specific thing.

Reply:
@kosuke55 
Yes that's the reason naraming the file!
> But in this case, since start and goal planner uses common utils, he need to put in in behavior_path_planner_common package I guess. So he rename the utils file name to show that the utils file is NOT module specific thing.

But if you have other idea, very welcome!